.github/workflows/Nightly_Func_Env_CI.yml

```diff
@@ -4,13 +4,13 @@ name: Deploy Func Env Nightly CI
 
 on:
   # Run on Nightly
-  schedule:
-    - cron: '0 4 * * 1-6' # run Monday-Saturday at 4 AM UTC/ 0 AM EDT
-  # Run on Sunday after assisted installer and operator completed
-  workflow_run:
-    workflows: ["Deploy Operator FUNC Env Weekly CI"]
-    types:
-      - completed
+#  schedule:
+#    - cron: '0 4 * * 1-6' # run Monday-Saturday at 4 AM UTC/ 0 AM EDT
+#  # Run on Sunday after assisted installer and operator completed
+#  workflow_run:
+#    workflows: ["Deploy Operator FUNC Env Weekly CI"]
+#    types:
+#      - completed
```

Comment:
Should these (and all of the other functional environment actions) go away altogether?

Reply:
Yes, we disable the cron job and calling by other workflow

Reply:
My question is whether we should remove these files.

Reply:
We can remove it but I dont see any problem to keep it as is

Reply:
Just to clean up unused code.  If we think we may want to bring it back, OK, but I'd like to keep everything tight.  I'm going to approve it; you decide whether you want to do that or not.